src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx

```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
 
 For Kafka running on Kubernetes, see [the Kubernetes requirements](http://docs.newrelic.com/docs/monitor-service-running-kubernetes#requirements).
 
+## Prepare for the installation [#prepare]
+
+Kafka is a complex piece of software that is built as a distributed system. For this reason, you’ll need to ensure that the integration can contact all the required hosts and services so the data is collected correctly.
+
+### Autodiscovery [#autodiscovery]
+
+Given the distributed nature of Kafka, the actual number and list of brokers is usually not fixed by the configuration, and it is instead quite dynamic. For this reason, the Kafka integration offers two mechanisms to perform automatic discovery of the list of brokers in the cluster: Bootstrap and Zookeeper. The mechanism you use depends on the setup of the Kafka cluster being monitored.
+
+#### Bootstrap [#bootstrap]
+
+With the bootstrap mechanism, the integration uses a bootstrap broker to perform the autodiscovery. This is a broker whose address is well known and that will be asked for any other brokers it is aware of. The integration needs to be able to contact this broker in the address provided in the bootstrap_broker_host parameter for bootstrap discovery to work.
+
+#### Zookeeper [#zookeeper]
+
+Alternatively, the Kafta integration can also talk to a Zookeeper server in order to obtain the list of brokers. To do this, the integration needs to be provided with the following:
+
+The list of Zookeeper hosts to contact (zookeeper_hosts).
+The proper authentication secrets to connect with the hosts.
```

Comment:
This is a bullet in the original doc.

Reply:
@urbiz-nr, I converted this to a bullet!